src/test/java/org/irods/operations/ResourceOperationsTest.java

```diff
@@ -3,9 +3,9 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.irods.IrodsHttpClient;
-import org.irods.properties.DataObject.DataObjectRemoveParams;
-import org.irods.properties.DataObject.DataObjectWriteParams;
-import org.irods.properties.Resource.ResourceCreateParams;
+import org.irods.properties.dataObject.DataObjectRemoveParams;
```

Comment:
Is `dataObject` a package or class?

Reply:
It's a package that has the params classes.

Reply:
Ok. According to https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html, `dataObject` should be `data_object`.

Let's do that for all package names which use two or more words.

Reply:
That is, use underscores to separate multiple words in package name elements.

Reply:
Ohh, I understand, makes sense. Done.